parameterized/test.py

```diff
@@ -294,9 +307,41 @@ def test_parameterized_argument_value_pairs(func_params, p, expected):
 def test_short_repr(input, expected, n=6):
     assert_equal(short_repr(input, n=n), expected)
 
+
 @parameterized([
-    ("foo", ),
+    ("foo",),
 ])
 def test_with_docstring(input):
     """ Docstring! """
     pass
+
+
+@parameterized.parameterized_class(('a', 'b', 'c'), [
+    ("", param(), []),
+    ("*a, **kw", param(), []),
+    ("*a, **kw", param(1, foo=42), [("*a", (1,)), ("**kw", {"foo": 42})]),
+    ("foo", param(1), [("foo", 1)]),
+    ("foo, *a", param(1), [("foo", 1)]),
+    ("foo, *a", param(1, 9), [("foo", 1), ("*a", (9,))]),
+    ("foo, *a, **kw", param(1, bar=9), [("foo", 1), ("**kw", {"bar": 9})]),
+    ("x=9", param(), [("x", 9)]),
+    ("x=9", param(1), [("x", 1)]),
+    ("x, y=9, *a, **kw", param(1), [("x", 1), ("y", 9)]),
+    ("x, y=9, *a, **kw", param(1, 2), [("x", 1), ("y", 2)]),
+    ("x, y=9, *a, **kw", param(1, 2, 3), [("x", 1), ("y", 2), ("*a", (3,))]),
+    ("x, y=9, *a, **kw", param(1, y=2), [("x", 1), ("y", 2)]),
+    ("x, y=9, *a, **kw", param(1, z=2), [("x", 1), ("y", 9), ("**kw", {"z": 2})]),
+    ("x, y=9, *a, **kw", param(1, 2, 3, z=3), [("x", 1), ("y", 2), ("*a", (3,)), ("**kw", {"z": 3})]),
+])
```

Comment:
Very nice!

A couple minor things:
- Have a test where the first argument is not a string
- Add assertions that check that both test methods were actually called (there's an example of how to do this somewhere else in this file; see `teardown_module`, I think)